code/game/atoms.dm

```diff
@@ -694,9 +694,11 @@
 
 	if(custom_materials)
 		var/list/materials_list = list()
-		for(var/datum/material/current_material as anything in custom_materials)
+		for(var/custom_material as anything in custom_materials)
+			var/datum/material/current_material = GET_MATERIAL_REF(custom_material)
```

Comment:
You don't need to ``as anything`` with no path to check

Reply:
Just habit haha